src/applications/vre/28-8832/config/form.js

```diff
@@ -25,47 +32,57 @@ const formConfig = {
   title: '28-8832-planning-and-guidance',
   defaultDefinitions: {},
   chapters: {
-    veteranDetails: {
-      title: 'Personal Information',
-      pages: {
-        veteranInformation: {
-          path: 'first-name',
-          title: 'Personal Information - Page 1',
-          uiSchema: veteranInformation.uiSchema,
-          schema: veteranInformation.schema,
-        },
-      },
-    },
-    militaryHistory: {
-      title: 'MIlitary History',
+    statusSelection: {
+      title: 'Your Status',
       pages: {
-        militaryHistory: {
-          path: 'military-history',
-          title: 'Military History',
-          uiSchema: militaryHistory.uiSchema,
-          schema: militaryHistory.schema,
+        wizard: {
+          path: 'your-status',
+          title: 'Your Status',
+          uiSchema: statusSelection.uiSchema,
+          schema: statusSelection.schema,
         },
       },
     },
     dependentInformation: {
       title: 'Dependent Information',
       pages: {
         dependentInformation: {
+          depends: formData => isDependent(formData),
           path: 'dependent-information',
           title: 'Dependent Information',
           uiSchema: dependentInformation.uiSchema,
           schema: dependentInformation.schema,
         },
+        dependentAddress: {
+          depends: formData => isDependent(formData),
+          path: 'dependent-address',
+          title: 'Dependent Address',
+          uiSchema: dependentAddress.uiSchema,
+          schema: dependentAddress.schema,
+        },
       },
     },
-    additionalInformation: {
-      title: 'Additional Information',
+    veteranDetails: {
+      title: 'Service member or Veteran information',
       pages: {
-        additionalInformation: {
-          path: 'additional-information',
-          title: 'Additional Information',
-          uiSchema: additionalInformation.uiSchema,
-          schema: additionalInformation.schema,
+        veteranInformation: {
+          path: 'personal-information',
+          title: 'Personal Information - Page 1',
```

Comment:
Do we actually need ` Page 1` here? Seems there should be a page 2 following this, otherwise it's probably fine to remove.